x-pack/legacy/plugins/ml/public/overview/components/analytics_panel/analytics_panel.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment, useState, useEffect } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiEmptyPrompt,
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiSpacer,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { AnalyticsTable } from './table';
+import { getAnalyticsFactory } from '../../../data_frame_analytics/pages/analytics_management/services/analytics_service';
+import { DataFrameAnalyticsListRow } from '../../../data_frame_analytics/pages/analytics_management/components/analytics_list/common';
+// TODO: panels can be smaller when empty
+export const AnalyticsPanel: FC = () => {
+  const [analytics, setAnalytics] = useState<DataFrameAnalyticsListRow[]>([]);
+  const [errorMessage, setErrorMessage] = useState<any>(undefined);
```

Comment:
maybe provide an error interface or just get rid of any?

Reply:
Will add as a follow up in the meta issue 👍  😄